efel/units/units.json

```diff
@@ -0,0 +1,129 @@
+{
+    "ADP_peak_amplitude": "mV",
+    "ADP_peak_values": "mV",
+    "AHP_depth": "mV",
+    "AHP_depth_abs": "mV",
+    "AHP_depth_abs_slow": "mV",
+    "AHP_depth_diff": "mV",
+    "AHP_depth_from_peak": "mV",
+    "AHP_depth_last": "mV",
+    "AHP_depth_slow": "mV",
+    "AHP_slow_time": "constant",
+    "AHP_time_from_peak": "ms",
+    "AHP_time_from_peak_last": "mV",
+    "AP1_peak": "mV",
+    "AP2_AP1_begin_width_diff": "ms",
+    "AP2_AP1_diff": "mV",
+    "AP2_AP1_peak_diff": "mV",
+    "AP_Amplitude": "mV",
+    "AP_Amplitude_change": "constant",
+    "AP_amplitude_diff": "mV",
+    "AP_amplitude_from_voltagebase": "mV",
+    "AP_begin_time": "ms",
+    "AP_begin_voltage": "mV",
+    "AP_begin_width": "ms",
+    "AP_duration": "ms",
+    "AP_duration_change": "constant",
+    "AP_duration_half_width": "ms",
+    "AP_duration_half_width_change": "constant",
+    "AP_fall_rate": "V",
+    "AP_fall_rate_change": "constant",
+    "AP_fall_time": "ms",
+    "AP_peak_downstroke": "V",
+    "AP_peak_upstroke": "V",
+    "AP_phaseslope": "1",
+    "AP_phaseslope_AIS": "1",
+    "AP_rise_rate": "V",
+    "AP_rise_rate_change": "constant",
+    "AP_rise_time": "ms",
+    "AP_width": "ms",
+    "AP_width_between_threshold": "ms",
+    "BAC_maximum_voltage": "mV",
+    "BAC_width": "ms",
+    "BPAPAmplitudeLoc1": "mV",
+    "BPAPAmplitudeLoc2": "mV",
+    "BPAPHeightLoc1": "mV",
+    "BPAPHeightLoc2": "mV",
+    "ISI_CV": "constant",
+    "ISI_log_slope": "ms",
+    "ISI_log_slope_skip": "ms",
+    "ISI_semilog_slope": "ms",
+    "ISI_values": "ms",
+    "Spikecount": "constant",
+    "Spikecount_stimint": "constant",
+    "adaptation_index": "constant",
+    "adaptation_index_2": "constant",
+    "all_ISI_values": "Hz",
```

Comment:
Should be ms

Reply:
Error originates from docs, to be updated

https://efel.readthedocs.io/en/latest/eFeatures.html#libv5-all-isi-values-inv-first-isi-inv-second-isi-inv-third-isi-inv-fourth-isi-inv-fifth-isi-inv-last-isi

Reply:
Both docs and API are updated